### ISSUE TITLE
Slice Tool: Add display and adjust Slice number

### ISSUE THREAD

**Issue by Hovden** [OpenChemistry/tomviz#91]
Users want to be able to view the current slice number and allow input to view a specific slice number in the slice tool.

Suggested Solution: A slider bar and an editable textbox that displays the current slice number.

**Comment by mathturtle** [comment]
Is this the non-orthogonal slice or the orthogonal slice?  The orthogonal slice has this, but for the non-orthogonal slice this doesn't really make sense...  How do you define slice numbers when the slice can be any arbitrary plane through the image and the data it is colored by is interpolated from nearby points?

**Comment by mathturtle** [comment]
@Hovden is this taken care of by adding the point and normal vector for the slice plane?  What exactly do you want here?

**Comment by Hovden** [comment]
Sorry about the delayed response. Thanks for the persistence.

>Your solution with points on the plane and normal vector partially solves this issue (but is also better in other ways). Only if you are interested in this issue should you work on it, otherwise we can move this issue off the 0.7 milestone and focus on higher priorities.

Basically, the user interacts with the slicer such that it translates up and down along the normal vector. It would be nice to be able to have a slider that does the same behavior. To give a use case, in one paper, we sliced uniformly incrementally along a non-orthogonal direction taking snapshots at each step. From these published snapshots we were able to determine the structure.
